src/reference/asciidoc/dsl.adoc

```diff
@@ -742,6 +742,32 @@ public IntegrationFlow subscribersFlow() {
 You can achieve the same result with separate `IntegrationFlow` `@Bean` definitions, but we hope you find the sub-flow style of logic composition useful.
 We find that it results in shorter (and so more readable) code.
 
+Starting with version 5.3, a `BroadcastCapableChannel`-based `publishSubscribeChannel()` implementation is provided to configure sub-flow subscribers on the broker-backed message channels.
+For example we now can configure several subscribers as sub-flows on the `Jms.publishSubscribeChannel()`:
+
+====
+[source,java]
+----
+@Bean
+public BroadcastCapableChannel jmsPublishSubscribeChannel() {
+    return Jms.publishSubscribeChannel(jmsConnectionFactory())
```

Comment:
Shouldn't we show the cast here? Or is your intention to fix the TODO in `JmsTests`?

Reply:
Yeah... I thought about that, but that is going to be kinda breaking change for the current `5.3` which should be compatible with the current Spring Cloud which will support both `5.2` and `5.3`.

Ok. I'll show the cast and we will remove in the next version when generics will be resolved properly.